src/pages/Home/index.js

```diff
@@ -1,8 +1,7 @@
 import Button from 'components/common/Button';
 import useTranslation from 'hooks/useTranslation';
 import { useLogoutMutation } from 'services/auth/auth';
-
-import logo from 'assets/logo.svg';
+import MapView from 'components/Map/index';
```

Comment:
It's not necessary to declare this route as `/index`. It'll always take by default the index file of the folder. You could live it as `components/Map`

Reply:
Ok, I got it, I will change it right now.